### PR TITLE
feat(admin-panel): add marketing notification UI

### PR DESCRIPTION
**Pull request by UncleSamtoshi**

### CODE REVIEW

File: apps/admin-panel/app/notifications/page.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import NotificationTestSender from "../../components/notification/test-sender"
+import NotificationFilteredSender from "../../components/notification/filtered-sender"
+
+export default function NotificationsScreen() {
+  const [notification, setNotification] = useState<NotificationContent>({
+    localizedPushContents: [],
+    deepLink: undefined,
+  })
+
+  const notificationContainsEnglishContent = notification.localizedPushContents.some(
```
Comment (siddhart1o1) [inline apps/admin-panel/app/notifications/page.tsx:17]: Notifications can only be triggered if English is included ?

**Review by siddhart1o1 — APPROVED** [review]
lgtm.

File: apps/admin-panel/components/notification/languages.ts [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+  Pashto: "ps",
+  Amharic: "am",
+  Yoruba: "yo",
+  Igbo: "ig",
+  Swahili: "sw",
+  Hausa: "ha",
+  Zulu: "zu",
+}
+
+export const CountryCodes = [
```
Comment (siddhart1o1) [inline apps/admin-panel/components/notification/languages.ts:46]: Maybe we could also include the name of the country in future PRs? That would make it easier to find.

File: apps/admin-panel/app/notifications/page.tsx (hunk 1 above)
Comment (UncleSamtoshi) [reply apps/admin-panel/app/notifications/page.tsx:17]: yup at the moment

File: apps/admin-panel/components/notification/languages.ts (hunk 2 above)
Comment (UncleSamtoshi) [reply apps/admin-panel/components/notification/languages.ts:46]: Thats good idea! Will look into adding that in a future pr.

**Pull request merged**